test/integration/bundle/test_create.py

```diff
@@ -11,7 +11,7 @@
 
 @pytest.mark.parametrize(
     argnames="test_version",
-    argvalues=["0.0.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0", "1.7.0b1", "1.7.0", "1.8.0b1"],
+    argvalues=["0.0.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0", "1.7.0b1", "1.7.0"],
```

Comment:
We can't currently test `1.8.0b1` since we know this will fail due to the metadata issue in the versions that's published to PyPI. We need to add it back once we publish the next `dbt-postgres` beta.